package.json

```diff
@@ -21,18 +21,21 @@
   "author": "Scale3 Labs",
   "license": "Apache-2.0",
   "dependencies": {
-    "@langtrase/trace-attributes": "6.0.4",
+    "@langtrase/trace-attributes": "file:../langtrace-trace-attributes/src/typescript/langtrase-trace-attributes-6.0.5.tgz",
```

Comment:
nit

Reply:
Plan to change once we are ready for release. Trace attributes needs to be released thats why its local at the moment